README.md

```diff
@@ -22,18 +22,19 @@ And then execute:
 
     $ bundle
     
-Then add a `css/main.scss` file based on the following template, don't forget to include the frontmatter:
 
-```css
----
----
+## Usage
 
-@import "minima";
-```
+### Customization
+
+To override the default structure and style of minima, simply create the concerned directory at the root of your site, copy the file you wish to customize to that directory, and then edit the file.
+e.g., to override the [`_includes/head.html `](_includes/head.html) file to specify a custom style path, create an `_includes` directory, copy `_includes/head.html` from minima gem folder to `<yoursite>/_includes` and start editing that file.
 
-You can choose to override the [`_includes/head.html `](_includes/head.html) file to specify a custom style path.
+`css/main.scss` has now moved to a new directory, `assets/` within the theme gem itself. To customize that file, you'll have to create an `assets` directory at the root of your site and copy the entire `css/` over to `<your-site>/assets/`.
```

Comment:
Thanks for the input, but I think it was too much info crammed into a few lines.. not good when a confused user checks this section for an answer. So I altered it _a bit_. 

> The site's default CSS has now moved to a new place within the gem itself, `assets/css/main.scss`. To **override the default CSS**, the file has to exist at your site source. Do either of the following:
- Create a new instance of `main.scss` at site source. 
  - Create a new directory `css` at `<your-site>/assets/`
  - Create a new file `main.scss` at `<your-site>/assets/css/`
  - Add the frontmatter dashes, and
  - Add `@import "minima";`, to `<your-site>/assets/main.scss`
  - Add your custom CSS.
- Download the file from this repo
  - Create a new directory `css` at `<your-site>/assets/`
  - Create  a new file `main.scss` at `<your-site>/assets/css/`
  - Copy the contents at [assets/css/main.scss]
(https://github.com/jekyll/minima/blob/master/assets/css/main.scss) onto the `main.scss` you just created, and edit away!
- Copy directly from Minima 2.0 gem
  - Go to your local minima gem installation directory ( run `bundle show minima` to get the path to it ).
  - Copy the `css/` folder from there into `<your-site>/assets/`
  - Change whatever values you want, inside `<your-site>/assets/css/main.scss`